tests/wpt/metadata-css/cssom-1_dev/html/interfaces.htm.ini

```diff
@@ -204,3 +204,9 @@
   [PseudoElement interface: attribute usedStyle]
     expected: FAIL
 
+  [CSSMediaRule interface: existence and properties of interface object]
```

Comment:
These fail because the CSSOM spec (which this test is based on) says that `CSSMediaRule` inherits from `CSSGroupingRule`, but the conditionals spec says that `CSSMediaRule` inherits from `CSSConditionRule` which in turn inherits from `CSSGroupingRule`.

I guess we should update the test? Not sure how this works, since this is a cssom-1 test.